src/api/v1/statuses.ts

```diff
@@ -429,6 +470,24 @@ app.delete(
           excludeBaseUris: [new URL(c.req.url)],
         },
       );
+      const acceptedRelays = await db.query.relays.findMany({
+        where: eq(relays.state, "accepted"),
+        with: {
+          relayServerActor: true,
+        },
+      });
+      await fedCtx.sendActivity(
+        owner,
+        acceptedRelays.map((relay) => ({
+          id: new URL(relay.relayServerActor.iri),
+          inboxId: new URL(relay.relayServerActor.inboxUrl),
+        })),
+        activity,
+        {
+          preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.